Update MASVS v2 info (#694)
OWASP/owasp-masvs: README.md

- ⚡ [Contribute!](#how-to-contribute)
- 💥 [Play with our Crackmes](https://mas.owasp.org/crackmes)

<br>

> <span>🆕</span> <span style="color: #33CCCC;"> The MASVS 2.0.0 is already available as a spreadsheet. We're currently working on updating this page and the related documents. Learn more about the refactoring process [here](https://github.com/OWASP/owasp-masvs/discussions/categories/big-masvs-refactoring).</span>

Author: ```suggestion
<div style="background-color: orange; color: white; border-radius: 10px; padding: 10px;">
    <p>The MASVS 2.0.0 is already available as a spreadsheet. We're currently working on updating this page and the related documents. Learn more about the refactoring process [here](https://github.com/OWASP/owasp-masvs/discussions/categories/big-masvs-refactoring).</p>
    <p>[MASVS v2.0.0](https://docs.google.com/spreadsheets/d/1MZIvJ5Aze-zpyzLvQZVwyzF0bKWRPfnEd7nqFeH2PfA/edit?usp=sharing)</p>
</div>

```

> [MASVS v2.0.0](https://docs.google.com/spreadsheets/d/1MZIvJ5Aze-zpyzLvQZVwyzF0bKWRPfnEd7nqFeH2PfA/edit?usp=sharing)

Author: ```suggestion
```

> 

Author: ```suggestion
```

- ⚡ [Contribute!](#how-to-contribute)
- 💥 [Play with our Crackmes](https://mas.owasp.org/crackmes)

<br>

> 

Author: ```suggestion

!!! important MASVS v2.0
    The MASVS 2.0.0 is already available as a spreadsheet. We're currently working on updating this page and the related documents. Learn more about the refactoring process [here](https://github.com/OWASP/owasp-masvs/discussions/categories/big-masvs-refactoring).
    
    [Access the MASVS v2.0.0](https://docs.google.com/spreadsheets/d/1MZIvJ5Aze-zpyzLvQZVwyzF0bKWRPfnEd7nqFeH2PfA/edit?usp=sharing)

```
